rollup.config.js

```diff
@@ -25,34 +31,32 @@ const base = {
     globals(),
     replace({
       'process.env.NODE_ENV': JSON.stringify('development'),
-      __VERSION__: JSON.stringify(pkg.version)
+      'process.env.VERSION': JSON.stringify(pkg.version)
     }),
     sizeSnapshot({
       matchSnapshot,
-      snapshotPath: path.join(packagesPath, 'jss/.size-snapshot.json')
+      snapshotPath: './.size-snapshot.json'
```

Comment:
why would you change things like that, I just recently made them all consistently full paths, since in some cases relative path was not properly resolved

Reply:
Oh is it supposed to resolve relative to each package dir?

Reply:
Yes, before we got a `.size-snapshot.json` file in the root folder. That way we get an individual file for every package.